app/models/validation_request.rb

```diff
@@ -1,23 +1,362 @@
 # frozen_string_literal: true
 
 class ValidationRequest < ApplicationRecord
```

Comment:
I wonder if renaming this to `PlanningApplication::Validation::Request` would make sense? But that would end up being a fairly sizeable piece of work in itself, so probably better in another PR (that someone else can do).